core/src/main/java/brooklyn/catalog/internal/BasicBrooklynCatalog.java

```diff
@@ -986,7 +1045,12 @@ private DeploymentPlan makePlanFromYaml(String yaml) {
     }
     
     private CatalogItem<?,?> addItemDto(CatalogItemDtoAbstract<?, ?> itemDto, boolean forceUpdate) {
-        checkItemNotExists(itemDto, forceUpdate);
+        CatalogItem<?, ?> existingDto = checkItemIsDuplicateOrDisallowed(itemDto, true, forceUpdate);
+        if (existingDto!=null) {
+            // it's a duplicate, and not forced, just return it
+            log.trace("Using existing duplicate for catalog item {}", itemDto.getId());
```

Comment:
log.debug instead of trace: we're treating the caller's request as a no-op, so should log that we've done nothing on the caller's request. Or does this get called a lot by some automated mechanism?